pkg/standalone/standalone.go

```diff
@@ -164,7 +166,8 @@ func Init(runtimeVersion, dashboardVersion string, dockerNetwork string, slimMod
 		// If --slim installation is not requested, check if docker is installed.
 		conatinerRuntimeAvailable := utils.IsDockerInstalled() || utils.IsPodmanInstalled()
 		if !conatinerRuntimeAvailable {
-			return errors.New("could not connect to Docker. Docker may not be installed or running")
+			caser := cases.Title(language.English)
+			return fmt.Errorf("could not connect to %s. %s may not be installed or running", caser.String(containerRuntime), caser.String(containerRuntime))
```

Comment:
Do we need this? Based on the docs it seems that the two lines convert `containerRuntime` to Title-Case in English. It should be ok to display them as-is without adding two experimental libraries.

Reply:
@shubham1172 I can refactor it easily to print container runtime values as they are. However `"golang.org/x/text` is already a dependency of the CLI itself